utils/utils.go

```diff
@@ -87,3 +87,27 @@ func defaultGOPATH() string {
 	}
 	return ""
 }
+
+func GetModule() bool {
```

Comment:
warning: exported function GetModule should have comment or be unexported (golint)